gdraw/gresedit.c

```diff
@@ -1020,7 +1020,7 @@ static void GResEditDlg(GResInfo *all,const char *def_res_file,void (*change_res
 	cnt = 0;
 	if ( res->extras!=NULL )
 	    for ( extras=res->extras, cnt = 0; extras->name!=NULL; ++cnt, ++extras );
-	tofree[i].earray = calloc(cnt+1,sizeof(GGadgetCreateData[8]));
+	tofree[i].earray = calloc(cnt+1,sizeof(GGadgetCreateData*[8]));
```

Comment:
I think this change is correct but my confidence is medium-level.

Reply:
This change is correct (not by testing but by further inspection). earray is doubled indexed, with the second ranging from 0..7, and stores pointers.